assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -545,6 +662,21 @@ const baseSelectors = {
 		// update.
 		return state.isAwaitingModulesRefresh;
 	} ),
+
+	canActivateModule( state, slug ) {
+		const moduleRequirements = state.checkRequirementsResults?.[slug];
```

Comment:
I don't think this is necessary here:
```suggestion
		const moduleRequirements = state.checkRequirementsResults[ slug ];
```

Reply:
Addressed.